src/content/docs/en/recipes/docker.mdx

```diff
@@ -148,10 +148,10 @@ WORKDIR /app
 COPY package.json package-lock.json ./
 
 FROM base AS prod-deps
-RUN npm install --production
+RUN npm install --omit=dev
 
 FROM base AS build-deps
-RUN npm install --production=false
+RUN npm install
```

Comment:
Suggestion: Instead of `install`, it would be safer to use `ci`. At this stage, installing is essentially a clean install, but if someone copies this into a different context, they might face some unintended behaviors.

``` suggestion
RUN npm ci
```

Reply:
I will keep this consistent with the original instruction, just removing the flag. If there are changes to be made to the recipe otherwise, we can always revisit!